vms/vm-under-test/scripts/customize-vm

```diff
@@ -90,21 +90,24 @@ cat <<'EOF' > "$checkup_boot_script_full_path"
 # Copyright 2024 Red Hat, Inc.
 #
 
-set -e
+set -x
 
 checkup_boot_ready_marker_full_path=$1
+checkup_tuned_adm_set_marker_full_path="/var/realtime-checkup-tuned-adm-set-marker"
 
 if systemctl --type swap list-units | grep -q '.swap'; then
   systemctl mask "$(systemctl --type swap list-units | grep '.swap' | awk '{print $1}')"
 fi
 
-tuned_conf="/etc/tuned/realtime-virtual-guest-variables.conf"
-echo "isolated_cores=2-3" > "$tuned_conf"
-echo "isolate_managed_irq=Y" >> "$tuned_conf"
-tuned-adm profile realtime-virtual-guest
+if [ ! -f "$checkup_tuned_adm_set_marker_full_path" ]; then
```

Comment:
Please consider reversing the condition so the script will `exist 0` in case the file exists.

By doing so, one indentation level will be eliminated.

Reply:
you still need to set the final flag (in the next commit) so you won't gain removing the indentation...
This way however keeps it consistent with the swap condition, so I think it's better..